packages/yuudachi/locales/en-US/translation.json

```diff
@@ -48,7 +48,23 @@
 					"no_reference_case": "Cannot find reference case `#{{case}}`",
 					"already_processed": "{{- user}} with case {{- case}} has no running restrictions.",
 					"no_reference_report": "Cannot find reference report `#{{report}}`"
-				}
+				},
+				"reasons": [
+					"Coppa",
```

Comment:
COPPA is an acronym, so this should probably be uppercased to be consistent with the other acronyms here.

Reply:
Fixed in 68523ac